pkgs/tools/X11/xrectsel/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, autoconf, automake, libX11 }:
+
+stdenv.mkDerivation rec {
+  name = "xrectsel-${version}";
+  version = "0.3.2";
+
+  src = fetchurl {
+    url = "https://github.com/lolilolicon/xrectsel/archive/${version}.tar.gz";
+    sha256 = "11g7zlbl80skbqfkvhgd1my4qy1c6a9py7kw8qqha3i7ny5r4jqv";
+  };
+
+  buildInputs = [ autoconf automake libX11 ];
```

Comment:
autoreconfHook in buildInputs probably makes your `preConfigure` step unnecessary, if it just do autotools stuff.